StandaloneMmPkg/Drivers/StandaloneMmIplPei/StandaloneMmIplPei.c

```diff
@@ -680,7 +721,7 @@ ExecuteMmCoreFromMmram (
   //
   Status = PeCoffLoaderGetImageInfo (&ImageContext);
   if (EFI_ERROR (Status)) {
-    return Status;
+    goto Done;

```

Comment:
The function returns `Status`. At this point, `Status` is some error. Are you expecting it to be overwritten by the status returned from `MmAccess->Close()` and `MmAccess->Lock()` in the `Done` block?